lib/takeaway.rb

```diff
@@ -0,0 +1,5 @@
+require 'dishes'
+
+class Takeaway
```

Comment:
I cant work out what this class represents, could you explain?
Could you refactor to remove this class completely?